### PR TITLE
assets/poll: remove popper styling and package and update poll styling

### PR DESCRIPTION
**Pull request by phillimorland**

### CODE REVIEW

File: meinberlin/assets/scss/components/_poll.scss [hunk 1]
```diff
@@ -187,14 +192,13 @@ $checkbox-size: 20px;
 }
 
 .editpoll__question-container {
-    // display: block;
+    display: flex;
```
Comment (khamui) [inline meinberlin/assets/scss/components/_poll.scss:195]: from screensize 992px to 991px it looks off

**Review by khamui — COMMENTED** [review]
one comment and frontend tests are failing (bc. of ErrorList location)

File: meinberlin/assets/scss/components/_poll.scss (hunk 1 above)
Comment (phillimorland) [reply meinberlin/assets/scss/components/_poll.scss:195]: @khamui having trouble replicating also that's 1px difference, is that the size you mean?

File: jest.config.js [hunk 2]
```diff
@@ -1,7 +1,11 @@
 const config = {
   testEnvironment: 'jsdom',
   modulePaths: [
-    '<rootDir>'
+    '<rootDir>',
+    '!venv'
```
Comment (goapunk) [inline jest.config.js:5]: why do we need this? we have the `modulePathIgnorePatterns` for it?

File: jest.config.js [hunk 3]
```diff
@@ -1,7 +1,11 @@
...
+  ],
+  modulePathIgnorePatterns: [
+    '<rootDir>/venv/*'
```
Comment (goapunk) [inline jest.config.js:8]: the asterisk is redundant here

File: meinberlin/assets/scss/components/_poll.scss (hunk 1 above)
Comment (khamui) [reply meinberlin/assets/scss/components/_poll.scss:195]: @phillimorland  i doublechecked and this occurs for me (in chrome), but not sure if that change was intended?

Bildschirmfoto 2022-10-24 um 13 37 04

before the questions were spanning all the way to the right edge?

Bildschirmfoto 2022-10-24 um 13 39 07

Comment (phillimorland) [reply meinberlin/assets/scss/components/_poll.scss:195]: cheers for screen shots, change wasn't intended, @goapunk is doing some jest config updates right now, will look at this once that's pushed

Comment (phillimorland) [reply meinberlin/assets/scss/components/_poll.scss:195]: @khamui ready for another look, can't tag you again

**Review by khamui — APPROVED** [review]
very nice! working great. Is it still depending on a4?
Leaving merge to @goapunk 

Question: do we need to update eslint and jest config in other places where we use them as well?

**Comment by goapunk** [comment]
> very nice! working great. Is it still depending on a4? Leaving merge to @goapunk
> 
> Question: do we need to update eslint and jest config in other places where we use them as well?

This already comes from a+ so at least there it's not necessary

**Comment by goapunk** [comment]
I bumped the a4 hash again so we don't have to do it again soon and set it to merge once CI passed

**Pull request merged**